Composer/packages/server/src/models/asset/assetManager.ts

```diff
@@ -234,4 +267,56 @@ export class AssetManager {
       }
     }
   }
+
+  private getFeedType(): FeedType {
+    // TODO: parse through data to detect for npm or nuget package schema and return respecive result
```

Comment:
I have code that does this in package manager.

Reply:
Ah thats right, I remember you mentioning that a while back. How shall we proceed? Want to send me the deets to that implementation and I can make the change to use that function?